Makefile

```
@@ -111,6 +125,7 @@ build-operator-image:
 build-operand-images:
 	$(IMAGE_BUILD_CMD) -f $(cert_manager_containerfile_name) -t $(CERT_MANAGER_IMAGE):$(IMAGE_VERSION) .
 	$(IMAGE_BUILD_CMD) -f $(cert_manager_acmesolver_containerfile_name) -t $(CERT_MANAGER_ACMESOLVER_IMAGE):$(IMAGE_VERSION) .
+	$(IMAGE_BUILD_CMD) -f $(istio_csr_containerfile_name) -t $(ISTIO_CSR_IMAGE):$(IMAGE_VERSION) .
```

nit: an improvement to existing code.
```suggestion
build-operand-images: build-cert-manager-image build-cert-manager-acmesolver-image build-istio-csr-image
```

---

Thanks for the suggestion added!!